_inc/client/components/chart/index.jsx

```diff
@@ -119,10 +120,7 @@ export default class ModuleChart extends React.Component {
 		if ( values.length && this.isEmptyChart( values ) ) {
 			emptyChart = (
 				<div className="dops-chart__empty">
-					<span className="dops-chart__empty_notice">
-						{ /* Translate this. */ }
-						No activity this period
-					</span>
+					<span className="dops-chart__empty_notice">{ __( 'No activity this period' ) }</span>
```

Comment:
Let's use context here please? Something like `_x( 'No activity this period', 'Notice in the empty statistics chart' )`.